css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Media Queries - Breakpoints</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      .card-wrapper {
+        padding: 0.5rem;
+        flex-basis: 100%;
+      }
+
+      /* your code here 👇 */
+      @media (min-width: 576px) {
```

Comment:
Add `only screen and` to the beginning of the media query

Reply:
Sure Thanks!